tests/security/cc/cc_selinux_setup.pm

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Setup "audit-test" test environment of a system running SELinux
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#93441
+
+use base 'selinuxtest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    my $selinux_config_file = "/etc/selinux/config";
```

Comment:
This variable seems not used, can we remove it now?

Reply:
> This variable seems not used, can we remove it now?

Done, good pointing, thanks :)